examples/rabbit_fox_hawk.jl

```diff
@@ -0,0 +1,428 @@
+# # 3D Predator-prey
+# ```@raw html
+# <video width="auto" controls autoplay loop>
+# <source src="../rabbit_fox_hawk.mp4" type="video/mp4">
+# </video>
+# ```
+# This model is a variation on the [Predator-Prey](@ref) example. It uses a 3-dimensional
+# [`GridSpace`](@ref), a realistic terrain for the agents, and pathfinding (with multiple
```

Comment:
ContinuousSpace now! (Also check that you don't say GridSpace in the rest of the example)